grow/commands/build.py

```diff
@@ -16,7 +16,8 @@
 @click.option('--clear_cache',
               default=False, is_flag=True,
               help='Clear the pod cache before building.')
-def build(pod_path, out_dir, preprocess, clear_cache):
+@click.option('--files', help='Build only pages affected by content files.', multiple=True)
```

Comment:
I think this should be `file` to be consistent with other flags - then you can do `--file=<> --file=<> --file=<>` etc.